public/apps/configuration/panels/role-edit/cluster-permission-panel.tsx

```diff
@@ -13,6 +13,7 @@
  *   permissions and limitations under the License.
  */
 
+import './_index.scss';
```

Comment:
Nonblocking. I thought this is done automatically. Looks like this is not the case for sub component. Do you know the reason?

Reply:
Actually not, I removed in previous commits. But when I do the sanity test today I found the layout differed.

Reply:
With import:
![image](https://user-images.githubusercontent.com/63078162/92666234-8b6ae280-f2bd-11ea-9443-1c024ec54fba.png)

Without import:
![image](https://user-images.githubusercontent.com/63078162/92666271-a63d5700-f2bd-11ea-87fa-c48fa53f8c56.png)

Reply:
Good catch. By the way, I meant to say if the component is the main component, e.g role edit, instead of a sub component which is used as part of another component.